Apps/phdossier/dossier.json

```diff
@@ -0,0 +1,3522 @@
+{
+    "appid": "341d81e0-5308-4d4d-93dd-d20a2a01d941",
+    "name": "Dossier",
+    "description": "This app supports investigate actions on the Infoblox Dossier platform",
+    "type": "information",
+    "product_vendor": "Infoblox",
+    "logo": "logo_dossier.svg",
+    "logo_dark": "logo_dossier_dark.svg",
+    "product_name": "Dossier",
+    "product_version_regex": ".*",
+    "publisher": "Nick Roy",
+    "license": "Copyright (c) Nick Roy, 2019",
+    "app_version": "1.0.1",
+    "utctime_updated": "2019-09-24T13:19:33.963396Z",
+    "package_name": "phantom_dossier",
+    "main_module": "dossier_connector.pyc",
+    "min_phantom_version": "4.5.15922",
+    "app_wizard_version": "1.0.0",
+    "configuration": {
+        "api_key": {
+            "description": "API Key for Dossier Service",
+            "data_type": "password",
+            "required": true,
+            "order": 0
+        }
+    },
+    "actions": [
+        {
+            "action": "test connectivity",
+            "identifier": "test_connectivity",
+            "description": "Validate the asset configuration for connectivity using supplied configuration",
+            "type": "test",
+            "read_only": true,
+            "parameters": {},
+            "output": [],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "lookup domain",
+            "identifier": "lookup_domain",
+            "description": "Check for the presence of a domain in a threat intelligence feed",
+            "type": "investigate",
+            "read_only": true,
+            "parameters": {
+                "domain": {
+                    "description": "Domain to lookup",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "domain",
+                        "url"
+                    ],
+                    "order": 0
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.domain",
+                    "data_type": "string",
+                    "example_values": [
+                        "twooo.cn",
+                        "sync.bfmio.com."
```

Comment:
These public domains should be obfuscated.

Reply:
Fixed.